Content.Client/GameObjects/Components/MagicMirrorBoundUserInterface.cs

```diff
@@ -256,10 +261,139 @@ public ColorSlider(string styleClass)
         }
     }
 
+    public class EyeColorPicker : Control
+    {
+        public event Action<Color> OnEyeColorPicked;
+
+        private readonly ColorSlider _colorSliderR;
+        private readonly ColorSlider _colorSliderG;
+        private readonly ColorSlider _colorSliderB;
+
+        private Color _lastColor;
+
+        public void SetData(Color color)
+        {
+            _lastColor = color;
+
+            _colorSliderR.ColorValue = color.RByte;
+            _colorSliderG.ColorValue = color.GByte;
+            _colorSliderB.ColorValue = color.BByte;
+        }
+
+        public EyeColorPicker()
+        {
+            var vBox = new VBoxContainer();
+            AddChild(vBox);
+
+            vBox.AddChild(_colorSliderR = new ColorSlider(StyleNano.StyleClassSliderRed));
+            vBox.AddChild(_colorSliderG = new ColorSlider(StyleNano.StyleClassSliderGreen));
+            vBox.AddChild(_colorSliderB = new ColorSlider(StyleNano.StyleClassSliderBlue));
+
+            Action colorValueChanged = ColorValueChanged;
+            _colorSliderR.OnValueChanged += colorValueChanged;
+            _colorSliderG.OnValueChanged += colorValueChanged;
+            _colorSliderB.OnValueChanged += colorValueChanged;
+        }
+
+        private void ColorValueChanged()
+        {
+            var newColor = new Color(
+                _colorSliderR.ColorValue,
+                _colorSliderG.ColorValue,
+                _colorSliderB.ColorValue
+            );
+
+            OnEyeColorPicked?.Invoke(newColor);
+
+            _lastColor = newColor;
+        }
+
+        private sealed class ColorSlider : Control
```

Comment:
Move `ColorSlider` out of `MagicMirrorBoundUserInterface.cs` and use the same instance instead of copy pasting it like this, please.